pipenv/patched/notpip/_internal/build_env.py

```diff
@@ -70,22 +68,18 @@ def _create_standalone_pip() -> Iterator[str]:
 
 
 class BuildEnvironment:
-    """Creates and manages an isolated environment to install build deps
-    """
+    """Creates and manages an isolated environment to install build deps"""
```

Comment:
PyCharm white space changes ... this makes an already huge PR even harder to review.
Please only change what needs to be changes without touching the white spae.

Reply:
I don't think this would be a PyChahrm white space space as I didn't even use pycharm to produce this PR.  I used the invoke vendoring commands.  See how it matches what is in 22.0.4, so this is actually correct and contains no actual changes introduced by me.  https://github.com/pypa/pip/blob/22.0.4/src/pip/_internal/build_env.py#L71